<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_addclass</class>
 <widget class="QDialog" name="dlg_addclass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="finish">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>131</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="name">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>课程名称</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="teacher">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>课程老师</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="location">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>上课地点</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="time">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>上课时间</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="test_time">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>考试时间</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="test_loc">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>考试地点</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="group">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>课程群</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>261</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLineEdit" name="le_name"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="le_teacher"/>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="le_loc"/>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="le_time"/>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="le_testtime"/>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="le_testloc"/>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>finish</sender>
   <signal>accepted()</signal>
   <receiver>dlg_addclass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finish</sender>
   <signal>rejected()</signal>
   <receiver>dlg_addclass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
